packages/http-client/__tests__/basics.test.ts

```diff
@@ -311,63 +313,63 @@ describe('basics', () => {
   it('posts a json object', async () => {
     const res = {name: 'foo'}
     const restRes = await _http.postJson<HttpBinData>(
-      'https://httpbin.org/post',
+      'https://postman-echo.com/post',
       res
     )
     expect(restRes.statusCode).toBe(200)
     expect(restRes.result).toBeDefined()
-    expect(restRes.result?.url).toBe('https://httpbin.org/post')
+    expect(restRes.result?.url).toBe('https://postman-echo.com/post')
     expect(restRes.result?.json.name).toBe('foo')
-    expect(restRes.result?.headers['Accept']).toBe(
+    expect(restRes.result?.headers[httpm.Headers.Accept]).toBe(
       httpm.MediaTypes.ApplicationJson
     )
-    expect(restRes.result?.headers['Content-Type']).toBe(
+    expect(restRes.result?.headers['content-type']).toBe(
       httpm.MediaTypes.ApplicationJson
     )
-    expect(restRes.headers[httpm.Headers.ContentType]).toBe(
+    expect(restRes.headers['content-type']).toContain(
       httpm.MediaTypes.ApplicationJson
     )
   })
 
   it('puts a json object', async () => {
     const res = {name: 'foo'}
     const restRes = await _http.putJson<HttpBinData>(
-      'https://httpbin.org/put',
+      'https://postman-echo.com/put',
       res
     )
     expect(restRes.statusCode).toBe(200)
     expect(restRes.result).toBeDefined()
-    expect(restRes.result?.url).toBe('https://httpbin.org/put')
+    expect(restRes.result?.url).toBe('https://postman-echo.com/put')
     expect(restRes.result?.json.name).toBe('foo')
 
-    expect(restRes.result?.headers['Accept']).toBe(
+    expect(restRes.result?.headers[httpm.Headers.Accept]).toBe(
       httpm.MediaTypes.ApplicationJson
     )
-    expect(restRes.result?.headers['Content-Type']).toBe(
+    expect(restRes.result?.headers['content-type']).toBe(
```

Comment:
nit: isn't the header key `Content-Type`? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type 

Is postman returning it lower case?